starknet_py/tests/e2e/client/fixtures/transactions.py

```diff
@@ -22,13 +22,18 @@
 
 @pytest_asyncio.fixture(scope="package")
 async def deploy_account_transaction(
-    account_with_validate_deploy_class_hash: int, fee_contract: Contract, network: str
+    account_with_validate_deploy_class_hash: int,
+    fee_contract: Contract,
```

Comment:
```suggestion
    eth_fee_contract: Contract,
```

Reply:
Thanks! I will commit it separately as it requires modifications in several other places.